src/cli.ts

```diff
@@ -168,7 +168,7 @@ program
   .requiredOption('--url <url>', 'kibana URL to upload the monitors')
   .requiredOption(
     '--auth <auth>',
-    'user authentication (in user:password format) or API key that will be used to communicate with Kibana.'
+    'API key used for Kibana authentication(https://www.elastic.co/guide/en/kibana/master/api-keys.html).'
```

Comment:
It seems like we should link to the specific project monitors docs, with a section about generating your api key, rather than the generic documentation. Going through Uptime to create your api key will ensure that the api key they create has the appropriate permissions. In fact, we don't have a good story for creating an api key outside of uptime, because the api key management page doesn't currently support api keys for kibana permissions, but our uptime implementation does.

Reply:
Good point, Would we have those docs available post 8.3? I am happy to do a followup.